docker-compose.yml

```diff
@@ -41,10 +46,15 @@ services:
   db:
     image: mariadb:10.8.2
     command: "mysqld --character-set-server=utf8 --collation-server=utf8_general_ci"
+    healthcheck:
+      test: "mysql --user=${OMRS_DB_USER:-openmrs} --password=${OMRS_DB_PASSWORD:-openmrs} --execute \"SHOW DATABASES;\""
```

Comment:
It might be better to use the `mysqladmin ping` command, no? This would fail if the openmrs user just doesn't have permissions to list the available databases.

Reply:
mysqladmin ping sometimes gives false positive, e.g. "This is 0 even in case of an error such as Access denied, because this means that the server is running but refused the connection, which is different from the server not running. ", see https://dev.mysql.com/doc/refman/8.0/en/mysqladmin.html. The permission issue should not be the case here, because we setup the DB with this user, but if it's a concern then I can just do "SELECT 1;" or something like that. It's more bulletproof.